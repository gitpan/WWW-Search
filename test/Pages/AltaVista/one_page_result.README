This set of test-result pages was created on Tue Jul 13 11:04:56 1999
