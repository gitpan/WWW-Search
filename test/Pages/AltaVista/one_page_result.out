http://www.isi.edu/div7/lsam/tools/autosearch/index.html
http://theory.uwinnipeg.ca/CPAN/data/WWW-Search/bin/AutoSearch.html
http://www.isi.edu/lsam/tools/autosearch/TEST/index.html
http://www.isi.edu/people/johnh/WORK/index.html
http://gost.isi.edu/people/johnh/WORK/index.html
http://www.isi.edu/~wls/
http://www.isi.edu/div7/lsam/tools/WWW_SEARCH/index.html
